java/openjdk-jdk.xml

```diff
@@ -20,6 +20,9 @@ IcedTea project.</description>
     <command name="java">
       <runner interface="http://repo.roscidus.com/java/openjdk-jre"/>
     </command>
+    <command name="java-gui">
+      <runner interface="http://repo.roscidus.com/java/openjdk-jre" command="run-gui"/>
+    </command>
```

Comment:
I don't think so, since there is no GUI version of `javac`.

The result I'm hoping to achieve is:

| Command | Effect on Windows | Effect on Linux |
|---|---|---|
| `0launch http://repo.roscidus.com/java/openjdk-jre` | `java` from JRE | `java` from JRE |
| `0launch --command=run-gui http://repo.roscidus.com/java/openjdk-jre` | `javaw` from JRE | `java` from JRE |
| `0launch http://repo.roscidus.com/java/openjdk-jdk` | `javac` from JDK | `javac` from JDK |
| `0launch --command=java http://repo.roscidus.com/java/openjdk-jdk` | `java` from JDK | `java` from JRE |
| `0launch --command=java-gui http://repo.roscidus.com/java/openjdk-jre` | `javaw` from JDK | `java` from JRE |